vector/src/main/java/im/vector/activity/VectorHomeActivity.java

```diff
@@ -287,32 +285,55 @@ public void onClick(DialogInterface dialog, int which) {
         final Intent intent = getIntent();
 
         if (intent.hasExtra(EXTRA_CALL_SESSION_ID) && intent.hasExtra(EXTRA_CALL_ID)) {
-            startCall(intent.getStringExtra(EXTRA_CALL_SESSION_ID), intent.getStringExtra(EXTRA_CALL_ID), (MXUsersDevicesMap<MXDeviceInfo>)intent.getSerializableExtra(EXTRA_CALL_UNKNOWN_DEVICES));
+            // fix issue #1276
+            // if there is a saved instance, it means that onSaveInstanceState has been called.
+            // theses parameters must only be used at activity creation.
+            // The activity might have been created after being killed by android while the application is in background
+            if (null == savedInstanceState) {
+                startCall(intent.getStringExtra(EXTRA_CALL_SESSION_ID), intent.getStringExtra(EXTRA_CALL_ID), (MXUsersDevicesMap<MXDeviceInfo>) intent.getSerializableExtra(EXTRA_CALL_UNKNOWN_DEVICES));
+            } else {
+                Log.d(LOG_TAG, "## onCreate() : ignore call params because null != savedInstanceState");
+            }
             intent.removeExtra(EXTRA_CALL_SESSION_ID);
             intent.removeExtra(EXTRA_CALL_ID);
             intent.removeExtra(EXTRA_CALL_UNKNOWN_DEVICES);
         }
 
         // the activity could be started with a spinner
         // because there is a pending action (like universalLink processing)
-        if (intent.getBooleanExtra(EXTRA_WAITING_VIEW_STATUS, WAITING_VIEW_STOP)) {
-            showWaitingView();
+        // fix issue #1276
+        // if there is a saved instance, it means that onSaveInstanceState has been called.
+        // theses parameters must only be used at activity creation.
+        // The activity might have been created after being killed by android while the application is in background
+        if (null == savedInstanceState) {
+            if (intent.getBooleanExtra(EXTRA_WAITING_VIEW_STATUS, WAITING_VIEW_STOP)) {
+                showWaitingView();
+            } else {
+                stopWaitingView();
+            }
         } else {
-            stopWaitingView();
+            Log.d(LOG_TAG, "## onCreate() : ignore EXTRA_WAITING_VIEW_STATUS because null != savedInstanceState");
         }
         intent.removeExtra(EXTRA_WAITING_VIEW_STATUS);
 
-        mAutomaticallyOpenedRoomParams = (Map<String, Object>) intent.getSerializableExtra(EXTRA_JUMP_TO_ROOM_PARAMS);
-        intent.removeExtra(EXTRA_JUMP_TO_ROOM_PARAMS);
+        // fix issue #1276
+        // if there is a saved instance, it means that onSaveInstanceState has been called.
+        // theses parameters must only be used at activity creation.
+        // The activity might have been created after being killed by android while the application is in background
+        if (null == savedInstanceState) {
```

Comment:
Maybe some refactoring could be done to avoid so many  `if (null == savedInstanceState)`. It would make it easier to maintain.